modules/core/src/main/java/org/apache/ignite/internal/processors/cache/IgniteCacheOffheapManagerImpl.java

```diff
@@ -3374,81 +3235,16 @@ private MvccUpdateTxStateHintHandler(CacheGroupContext grp) {
      * Applies changes to the row.
      */
     private static final class MvccApplyChangesHandler extends PageHandler<MvccDataRow, Boolean> {
-        /** */
-        private final CacheGroupContext grp;
-
         /** */
         private MvccApplyChangesHandler(CacheGroupContext grp) {
-            this.grp = grp;
         }
 
         /** {@inheritDoc} */
         @Override public Boolean run(int cacheId, long pageId, long page, long pageAddr, PageIO io, Boolean walPlc,
             MvccDataRow newRow, int itemId, IoStatisticsHolder statHolder) throws IgniteCheckedException {
-            assert grp.mvccEnabled();
+            assert false; // ex mvcc code.
```

Comment:
Do we need this assert. // No-op. istead? Also, the exception throwing declaration might be removed